sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_attribution_v1/metadata.yaml

```diff
@@ -15,6 +15,7 @@ labels:
 scheduling:
   dag_name: bqetl_analytics_aggregations
   task_name: active_users_aggregates_attribution_v1
+  date_partition_offset: -1
```

Comment:
Offsets need to be added manually here since our machinery is not able to determine that upstream dependencies of upstream dependencies have a `date_partition_offset`.